py3status/modules/imap.py

```diff
@@ -1,83 +1,94 @@
 # -*- coding: utf-8 -*-
-# FIXME new_mail_color param
 """
 Display number of unread messages from IMAP account.
 
 Configuration parameters:
-    cache_timeout: how often to run this check (default 60)
+    cache_timeout: refresh interval for this module (default 60)
     criterion: status of emails to check for (default 'UNSEEN')
-    format: format to display (default 'Mail: {unseen}')
-    hide_if_zero: don't show on bar if True (default False)
-    imap_server: IMAP server to connect to (default '<IMAP_SERVER>')
+    format: display format for this module (default 'Mail: {unseen}')
+    hide_if_zero: hide this module when no new mail (default False)
     mailbox: name of the mailbox to check (default 'INBOX')
-    new_mail_color: what color to output on new mail (default '')
-    password: login password (default '<PASSWORD>')
-    port: IMAP server port (default '993')
-    security: what authentication method is used: 'ssl' or 'starttls'
+    password: login password (default None)
+    port: number to use (default '993')
+    security: login authentication method: 'ssl' or 'starttls'
         (startssl needs python 3.2 or later) (default 'ssl')
-    user: login user (default '<USERNAME>')
+    server: server to connect (default None)
+    user: login user (default None)
 
 Format placeholders:
     {unseen} number of unread emails
 
+Color options:
+    color_new_mail: use color when new mail arrives (default None)
+
 @author obb
 """
-
 import imaplib
 from ssl import create_default_context
+STRING_UNAVAILABLE = 'Mail: N/A'
 
 
 class Py3status:
     """
     """
     # available configuration parameters
     cache_timeout = 60
+    color_new_mail = None
```

Comment:
This setting should be removed.  `color_..` setting do leak to modules but we shouldn't default them or use them in the modules.  We happen to for backwards compatibility